include/range/v3/detail/config.hpp

```diff
@@ -26,12 +26,12 @@
 #define __has_builtin(x) 0
```

Comment:
Is this non-portable because of the __ or for other reasons?
If it is for __, it can be replaced by RANGE_HAS_BUILTIN and RANGE_BUILTIN_ASSUME.